### DISCUSSION TITLE
Renovate open several PR instead of one for js

### DISCUSSION THREAD

**Discussion by t-botz** [renovatebot/renovate#11827] [Request Help]
I have been using renovate with several languages and my usual config is:
```json5
{
  extends: ["config:base", "schedule:weekly"],
  groupName: "all",
  separateMajorMinor: false,
}
```
The goal being to have a single big PR weekly. 

I just did the same config in a Javascript and renovate separate PRs in a way that I dont get.
Some dependencies will fall in the `all` group (e.g `eslint-plugin-jest`, `graphql`, `prettier`) while other will have their own group like :
 - `Apollo GraphQL packages` (`@apollo/gateway`, `apollo-server-core`, ...)
 - `typescript-eslint monorepo` (`@typescript-eslint/eslint-plugin`, `@typescript-eslint/parser`)
 - Jest in its own group
 
 
 I dont understand the logic nor could I find the configuration to prevent this grouping.

**Comment by rarkins** [comment]
config:base has some presets included for grouping monorepos and other tightly related packages. You need to add these to ignorePresets or else stop using config:base

**Reply by t-botz** [reply]
That makes sense! Thanks!

**Reply by viceice** [reply]
You can use a packageRule to override all buildin groups from `config:base`:

```json
{
  "matchPackagePatterns": ["*"],
  "groupName": "all"
}
```

**Reply by t-botz** [reply]
@viceice based on your answer and the group preset i have just discovered, i guess I can replace my whole config with:

```json5
{
   extends: ["config:base", "group:all", "schedule:weekly"],
}
```

Isn't it?

**Reply by viceice** [reply]
should work
